lib/db_helper/tasks.rb

```diff
@@ -77,7 +77,7 @@
     hot_backup_dir = InputReader.get_string(prompt: "Backup path (#{default_hot_backup_dir}):", default_value: default_hot_backup_dir)
     raise "#{hot_backup_dir} is not a valid path" unless File.directory?(hot_backup_dir)
 
-    sandbox_proc = Sys::ProcTable.ps.find { |p| p.comm == 'mysqld' && 
+    sandbox_proc = Sys::ProcTable.ps.find { |p| p.comm == 'mysqld' &&
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Prefer `detect` over `find`.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.